src/components/ValidateCodeActionModal/index.tsx

```diff
@@ -61,10 +74,13 @@ function ValidateCodeActionModal({isVisible, title, description, onClose, valida
                         validatePendingAction={validatePendingAction}
                         validateError={validateError}
                         handleSubmitForm={handleSubmitForm}
+                        sendValidateCode={sendValidateCode}
```

Comment:
Is there any reason why we made this a prop instead of implementing it this component?

Reply:
Basically this code is repeated in every component that uses this modal

```
    const sendValidateCode = () => {
        if (loginData?.validateCodeSent) {
            return;
        }

        User.requestValidateCodeAction();
    };
```